elementary/monitor/dbt_project/macros/base_queries/current_tests_run_results_query.sql

```diff
@@ -186,6 +188,16 @@
 
     select
         test_results.id,
+        {#
```

Comment:
Moved `test_sub_type_unique_id` into the query and added support for the use cases of `schema changes` and `dimension anomalies`